src/Parsers/Access/parseUserName.h

```diff
@@ -34,4 +35,9 @@ inline bool parseRoleNames(IParser::Pos & pos, Expected & expected, Strings & ro
     return parseUserNames(pos, expected, role_names);
 }
 
+inline bool parseStorageName(IParser::Pos & pos, Expected & expected, String & storage_name)
```

Comment:
We have storages for tables too, so the name `parseStorageName()` can be a bit confusing. Let's rename it to `parseAccessStorageName()`. And probably it's also better to move it to its own header or maybe to `IAccessStorage.h`